base/shell/explorer/lang/en-US.rc

```diff
@@ -196,3 +196,8 @@ BEGIN
     IDS_TASKBAR_STARTMENU_PROP_CAPTION "Taskbar and Start Menu"
     IDS_RESTORE_ALL "&Show Open Windows"
 END
+STRINGTABLE
+BEGIN
+    IDS_TRAYWND_UNDO_CASCADE "Undo cascade windows"
```

Comment:
For these new ones (in all other translations than fr-fr) you could put keyboard accelerator on the "c" of cascade, and "t" of tile.

Reply:
Done !